datastore/db/queries/nutrients/__init__.py

```diff
@@ -0,0 +1,386 @@
+"""
+This module represent the function for the table micronutrient, guaranteed and its children element_compound:
+
+"""
+class ElementCreationError(Exception):
+    pass
+
+class ElementNotFoundError(Exception):
+    pass
+
+class MicronutrientCreationError(Exception):
+    pass
+
+class MicronutrientNotFoundError(Exception):
+    pass
+
+class GuaranteedCreationError(Exception):
+    pass
+
+class GuaranteedNotFoundError(Exception):
+    pass
+
+def new_element(cursor,number,name_fr,name_en,symbol):
+    """
+    This function add a new element in the database.
+
+    Parameters:
+    - cursor (cursor): The cursor of the database.
+    - number (int): The number of the element.
+    - name_fr (str): The name of the element in French.
```

Comment:
number is the number on the periodic table if its a simple element. but yes we should have it in both language.`Boron` and 'Bore' are the same thing but labeled differently. Also the client will most likely use the name of the element or its symbole. The number of the element is there because, why not have more information for the elements in the periodic table?

Reply:
So I guess we keep it for recovery (start from where we left) and comparison (before/after edit)?